scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/job/TaskDataSpaceApplication.java

```diff
@@ -49,21 +49,21 @@
 import org.ow2.proactive.scheduler.core.properties.PASchedulerProperties;
 
 
-public class JobDataSpaceApplication implements Serializable {
+public class TaskDataSpaceApplication implements Serializable {
 
-    public static final Logger logger = Logger.getLogger(JobDataSpaceApplication.class);
+    public static final Logger logger = Logger.getLogger(TaskDataSpaceApplication.class);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make the "logger" logger private static final and rename it to comply with the format "LOG(?:GER)?". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1312)